activiti-services/activiti-services-events/src/main/java/org/activiti/services/events/MessageProducerActivitiEventListener.java

```diff
@@ -24,10 +29,34 @@ public MessageProducerActivitiEventListener(ProcessEngineChannels producer,
 
     @Override
     public void onEvent(ActivitiEvent event) {
+        CommandContext currentCommandContext = Context.getCommandContext();
```

Comment:
I know that there is not unit test for this class yet, but I think that's right moment to do so. This static call to `Context.getCommandContext()` will prevent mocking the context. I'd suggest to extract this inside a `ContextWrapper` component that's is injected in the `MessageProducerActivitiEventListener `, so that you can mock it. What do you think?